scripts/nginx/sonarr.sh

```diff
@@ -50,11 +48,13 @@ fi
 
 apikey=$(grep -oPm1 "(?<=<ApiKey>)[^<]+" "$app_configdir"/config.xml)
 
+# Set to Debug as this is alpha software
+# ToDo: Logs back to Info
 cat > "$app_configdir"/config.xml << ARRCONFIG
 <Config>
-  <LogLevel>info</LogLevel>
+  <LogLevel>trace</LogLevel>
   <UpdateMechanism>BuiltIn</UpdateMechanism>
-  <BindAddress>127.0.0.1</BindAddress>
+  <BindAddress>localhost</BindAddress>
```

Comment:
why localhost?

ipv6 can sometimes get fucky

Reply:
given the numerous instances of ipv6 being enabled but non-functional, users' lack of awareness of ipv6, dotnet's medicore ipv6 support - don't see why.

Reply:
Agree. I would also like to see a 127.0.0.1 bind here.

It's possible this can manifest as connection refused errors, so for broad support I find it better to hardcode this.